### DISCUSSION TITLE
Can't scroll when editing a post. Screen colours have inverted[Bug report]

### DISCUSSION THREAD

**Discussion by VoxNumana** [GetPublii/Publii#1791] [Bug Report] [answered]
### Operating system

Win 11

### Publii version

0.46.2

### Issue type

Application

### Bug description

Updated to latest release. When I go to edit a post, the screen is black text on very dark grey background. Also, I cannot scroll when editing post

**Comment by bobmitro** [comment]
There is an issue loading the CSS for WYSIWYG  editor  on Windows only. We will fix it soon.

**Reply by VoxNumana** [reply, marked as answer]
Ok, thanks

**Comment by bobmitro** [comment]
Download the Windows version once again, specifically build 16968. This version fixes the problem.
